datadog_lambda/trigger.py

```diff
@@ -369,3 +367,28 @@ def extract_http_status_code_tag(trigger_tags, response):
         status_code = response.status_code
 
     return str(status_code)
+
+
+def is_step_function_event(event):
```

Comment:
Is there a way we can memoize this function? It looks like it can potentially be called several times in the course of a single invocation.

Reply:
Hmm, or it looks like the function can be called multiple times per invocation, but with different "events" each time? If that's true, then we can probably leave it.

Reply:
That's a great idea!

Correct me if I'm wrong but does the layer only handle one `event` per invocation? Or if it's a busy Lambda does it stay alive and potentially handle hundreds of events?

Just wondering to get an idea of how large to make the cache. I guess it can be pretty small anyway since each event is new and we don't repeat

Reply:
Each runtime instance will only ever handle one event at a time. It never handles two events concurrently.

Reply:
Ah just realized we can't memoize it because `event` is a `dict` and mutable types are unhashable

We could serialize the dict and use that but I'm thinking that'd be much slower